src/state/stableswap-add-liquidity/hooks.ts

```diff
@@ -42,21 +43,24 @@ export function useDerivedStableSwapAddLiquidityInfo(
     [Field.CURRENCY_0]: typedValue0,
     [Field.CURRENCY_1]: typedValue1,
     [Field.CURRENCY_2]: typedValue2,
-    [Field.CURRENCY_3]: typedValue3
+    [Field.CURRENCY_3]: typedValue3,
+    [Field.CURRENCY_4]: typedValue4
   } = useStableSwapAddLiquidityState()
 
   const { lpToken, poolTokens } = STABLESWAP_POOLS[stableSwapPoolName]
   const [currency0, currency1, currency2, currency3] = poolTokens.map(token => unwrappedToken(token))
   const hasThirdCurrency = currency2 != null
   const hasFourthCurrency = hasThirdCurrency && currency3 != null
+  const hasFifthCurrency = hasFourthCurrency && currency3 != null
```

Comment:
typo

Reply:
🙏🙏🙏

I was on a flight so didn't have time to test it #draftstatus

Reply:
Would've taken me forever to find hahah